server/src/controllers/client/addClients.js

```diff
@@ -0,0 +1,56 @@
+const logger = require("pino")();
+const Client = require("../../models/client.model");
+
+const addClientsRequestHandler = async (req, res) => {
```

Comment:
Given the recent model changes, we should be adding the date_created to each of the models.(Sequelize may provide the ability to auto set this attribute by adding something to the model definition)